DEVELOPMENT.md

```diff
@@ -24,7 +24,6 @@ From the project root, run the following commands
 - __Create a local build__ by running `pnpm build`, this will populate local dependencies
 - __Run dev mode__ by running `pnpm dev` or `pnpm dev:electron` to get the electron window
 
-
```

Comment:
nitpick: this diff doesnt seem to do anything, could we remove it to cleanup the change?